go.mod

```
@@ -1,6 +1,6 @@
 module github.com/kanopy-platform/go-http-middleware
 
-go 1.17
+go 1.24
```

This can go to 1.25 now.
```suggestion
go 1.25
```